CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.0] - 2020-09-22
+- bump miq-messaging version for rdkafka changes #TBD
```

Comment:
This should be #209, right? I don't want to block this PR due to comment, so we'll change it next time